yb-voyager/cmd/exportDataDebezium.go

```diff
@@ -405,10 +405,12 @@ func checkAndHandleSnapshotComplete(config *dbzm.Config, status *dbzm.ExportStat
 			}
 			if !(msr.ExportFromTargetFallBackStarted || msr.ExportFromTargetFallForwardStarted) {
 				utils.PrintAndLog("Waiting to initialize export of change data from target DB...")
-				// only events received after yb cdc initialization will be emitted by debezium.
-				// Therefore, we sleep to allow yb cdc connector to initialize and only then mark the cutover to be complete.
-				// Ideally, we should have a more reliable way to determine that init is complete. This is a temp solution.
-				time.Sleep(2 * time.Minute)
+				logFilePath := filepath.Join(exportDir, "logs", fmt.Sprintf("debezium-%s.log", exporterRole))
+
+				err := utils.WaitForLineInLogFile(logFilePath, "Beginning to poll the changes from the server")
```

Comment:
pls add a comment here explaining why we are doing this in the first place. I've added some context in the jira, see if that helps.